akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpEncoding.scala

```diff
@@ -55,7 +55,6 @@ object HttpEncodings extends ObjectRegistry[String, HttpEncoding] {
   val deflate          = register("deflate")
   val gzip             = register("gzip")
   val identity         = register("identity")
-  val `identity;q=MIN` = identity.withQValue(Float.MinPositiveValue)
```

Comment:
This `identity;q=MIN` thing was only a "convenience" thing anyway, with almost zero real-life usage, however.
`gzip` and `deflate` are the only things that really matter.